api/v1/verticadb_webhook.go

```diff
@@ -1773,30 +1773,28 @@ func (v *VerticaDB) checkNewSBoxOrSClusterShutdownUnset(allErrs field.ErrorList)
 // live in a sandbox where Shutdown is set to false in spec/status for all pieces
 func (v *VerticaDB) checkShutdownForScaleUpOrDown(oldObj *VerticaDB, allErrs field.ErrorList) field.ErrorList {
 	newSclusterMap := v.GenSubclusterMap()
-	newSclusterSboxMap := v.GenSubclusterSandboxMap()
-	newSandboxMap := v.GenSandboxMap()
-	statusSclusterIndexMap := v.GenStatusSClusterIndexMap()
 	oldSclusterMap := oldObj.GenSubclusterMap()
 	newSclusterIndexMap := v.GenSubclusterIndexMap()
-	for subclusterName, sandboxName := range newSclusterSboxMap {
+	statusSubclusterMap := v.GenStatusSubclusterMap()
+	sclusterSboxMap := v.GenSubclusterSandboxMap()
+	for subclusterName, newScluster := range newSclusterMap {
 		oldScluster, foundSclusterInOld := oldSclusterMap[subclusterName]
 		if !foundSclusterInOld {
 			continue // we only need to deal with persistent subclusters
 		}
-		newScluster, foundSclusterInNew := newSclusterMap[subclusterName]
-		if !foundSclusterInNew {
-			continue // avoid unexpected panic when something is wrong
+		_, inSandbox := sclusterSboxMap[subclusterName]
+		if inSandbox {
+			continue // this scenario is handled by another webhook rule
```

Comment:
Which rule handles this one? If you don't check the subclusters in sandboxes, you need to remove comment in line 1773

Reply:
line 1631.
// checkImmutableSubclusterInSandbox ensures we do not scale and remove any subcluster that is in a sandbox
func (v *VerticaDB) checkImmutableSubclusterInSandbox(oldObj *VerticaDB, allErrs field.ErrorList) field.ErrorList {

comment is removed.

Reply:
Minor: update this line's comment to "this scenario is handled by checkImmutableSubclusterInSandbox()" in your next PR.